### PR TITLE
feat: icon picker component #P23-215

### PR DESCRIPTION
**Pull request by aniasgit**

### CODE REVIEW

File: packages/ui/IconPicker/iconPicker.styled.tsx [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  grid-template-columns: repeat(3, 64px);
+  grid-template-rows: repeat(3, 64px);
+  width: fit-content;
+  height: fit-content;
+  padding: 4px;
+`;
+
+export const SelectIconButtonStyled = styled.button`
+  display: flex;
+  justify-content: center;
```
Comment (Noshi96) [inline packages/ui/IconPicker/iconPicker.styled.tsx:62]: You can add ```cursor: pointer;``` also for internal icon buttons

Comment (aniasgit) [reply packages/ui/IconPicker/iconPicker.styled.tsx:62]: You're right 😄

File: packages/ui/IconPicker/index.tsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+  IconPickerStyled,
+  IconAndButtonWrapperStyled,
+  IconsSelectorStyled,
+  EditButtonStyled,
+  SelectIconButtonStyled,
+} from "./iconPicker.styled";
+import { Icon, IconType } from "../Icon";
+
+type IconPickerProps = {
+  defaultIcon?: IconType;
```
Comment (Noshi96) [inline packages/ui/IconPicker/index.tsx:13]: I think it is better to make the defaultIcon as a mandatory property.

Comment (monika-klokosinska-intive) [reply packages/ui/IconPicker/index.tsx:13]: @Noshi96 @aniasgit as the creator of this task I assumed there might be a use case where the user hasn't selected anything yet and we want to reflect that on the UI.

File: apps/storybook/stories/IconPicker.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
+import React, { useState } from "react";
```
[the remark this answers is not in the document]
Comment (aniasgit) [reply apps/storybook/stories/IconPicker.stories.tsx:1]: If we don't import React in storybook, there are some problems: 
Bez tytułu2
Everybody use import React in storybook

[the remark this answers is not in the document]
Comment (monika-klokosinska-intive) [reply apps/storybook/stories/IconPicker.stories.tsx:1]: @aniasgit Makes sense, thanks for the clarification!

**Review by monika-klokosinska-intive — APPROVED** [review]

File: packages/ui/IconPicker/iconPicker.styled.tsx (hunk 1 above)
Comment (aniasgit) [reply packages/ui/IconPicker/iconPicker.styled.tsx:62]: it's done

**Review by Noshi96 — APPROVED** [review]

**Review by Noshi96 — APPROVED** [review]

**Review by bdeluga — APPROVED** [review]
Looks, very good ! 👀

**Comment by monika-klokosinska-intive** [comment]
@aniasgit I have one accessibility improvement to make - when the picker element with 9 icons is visible, pressing escape on the keyboard should close the picker. 

Great work otherwise!

**Comment by aniasgit** [comment]
> @aniasgit I have one accessibility improvement to make - when the picker element with 9 icons is visible, pressing escape on the keyboard should close the picker.
> 
> Great work otherwise!

Good idea! I've just added it :)

**Review by monika-klokosinska-intive — APPROVED** [review]

**Pull request merged**